### ISSUE TITLE
granubuf~, etc. causing crash on M1 Macs

### ISSUE THREAD

**Issue by wagne342** [ramagottfried/tilde#3]
Getting reports from teaching staff that students using granubuf~ on M1 Macs observe a crash when audio is turned on. Obviously, there isn't an arm64 version yet. I have cloned the repo to see what can be done on our end, but the Xcode project points to files that aren't present in the repo, namely, the "include" folder

**Comment by ramagottfried** [comment]
Hi @wagne342, I have an M1 computer now, and am starting to work on this a bit. The include folder is in the src directory, when I was updating the Xcode setup, I had to update the folder paths for the new max-sdk, but was able to build and run granubuf.mc~, not sure about the signing process yet, maybe I can send you some objects to test?

**Comment by wagne342** [comment]
Absolutely! Send away. Converting the xcode is the tricky bit, but once that is done, it is usually smooth sailing.

**Comment by ramagottfried** [comment]
hi! sorry for the delay, super busy first week of school here -- ok, lets see if this works:
[granubuf.mc.m1.test.zip](https://github.com/ramagottfried/tilde/files/9671532/granubuf.mc.m1.test.zip)

this was built on my computer so maybe it will be quarantined on your computer? does it work otherwise? haven't tested on non m1 machines.
